### PR TITLE
Add while loop counter

### LINKED ISSUE #776: Counter for while loop
It would be nice to have a counter `n` for a while loop too.
[Try it online!](https://vyxal.pythonanywhere.com/#WyIiLCIiLCIxe24sIiwiIiwiIl0=)
So it would print 1, 2, 3, 4, 5, ...
Currently, it requires [7 bytes (I think)](https://vyxal.pythonanywhere.com/#WyIiLCIiLCIxwqN7wqXigKbigLrCoyIsIiIsIiJd)

### PR DESCRIPTION
**Pull request by SE-Nitro**
closes #776 
Now eg. `{n,` outputs:
```
1
2
3
...
```
Should the counter start at 1?

### CODE REVIEW

**Comment by Seggan** [comment]
> Should the counter start at 1?

 I vote 0

**Comment by TheSecondComing123** [comment]
I vote 0.

**Review by TheSecondComing123 — CHANGES_REQUESTED** [review]
I vote it start at 0.

**Comment by SE-Nitro** [comment]
[Poll](https://strawpoll.com/polls/bVg86b84zyY)

**Comment by SE-Nitro** [comment]
It would make sense to start with 1, because in vyxal [numbers also start with 1 when used as ranges (in a loop)](https://vyxal.pythonanywhere.com/#WyIiLCIiLCI1KG4sIiwiIiwiIl0=)

**Comment by SE-Nitro** [comment]
> 

So, you mean, start at 0 by default and start at 1 with flag `M`?

**Comment by chunkybanana** [comment]
0 because ( behaves like that.

**Comment by AMiller42** [comment]
> 0 because ( behaves like that.

[But it don't tho.](https://vyxal.pythonanywhere.com/#WyIiLCIiLCI2OShuLCIsIiIsIiJd)

**Comment by SE-Nitro** [comment]
Now, the default is `1` and `M` makes it `0`

**Comment by SE-Nitro** [comment]
> > 0 because ( behaves like that.
> 
> [But it don't tho.](https://vyxal.pythonanywhere.com/#WyIiLCIiLCI2OShuLCIsIiIsIiJd)

<strike>funny number</strike>

File: vyxal/main.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
     j    Print top of stack joined by newlines on end of execution
     L    Print top of stack joined by newlines (Vertically) on end of execution
     s    Sum/concatenate top of stack on end of execution
-    M    Make implicit range generation start at 0 instead of 1
+    M    Make implicit range generation start at 0 instead of 1 (and while loop counter too)
```
Comment (Seggan) [inline vyxal/main.py:29]: Maybe change to `M Make implicit range generation and the while loop counter start at 0 instead of 1

**Review by Seggan — CHANGES_REQUESTED** [review]

File: vyxal/main.py (hunk 1 above)
Comment (edison1119) [reply vyxal/main.py:29]: `Make implicit range generation and loop counter start at 0 instead of 1`

**Review by Seggan — APPROVED** [review]

**Review by TheSecondComing123 — APPROVED** [review]

**Pull request merged**